lib/hexpm/repository/package.ex

```diff
@@ -1,5 +1,6 @@
 defmodule Hexpm.Repository.Package do
   use Hexpm.Schema
+  alias Hexpm.Repository.PackageDependant
```

Comment:
Is there a reason I have to `alias` this while modules like `PackageDownload` don't need to be?

Reply:
Answered my own question.